comingsoon/index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta property="og:title" content="HackRU Fall 2017"/>
+    <meta property="og:description" content="Get hyped for the best hackathon that Rutgers has to offer! The 13th semiannual HackRU is coming this Fall 2017, from October 14th-15th at College Ave Student Center." />
+    <meta property="og:image" content="http://www.hackru.org/assets/hackRU_SP17_avatar.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Sign up for HackRU Fall 2017, taking place from October 14th to 15th.">
+    <title>HINT v3.0</title>
+
+    <link href="assets/fav.png" rel="shortcut icon" type="image/png"/>
+
+    <link rel="stylesheet" type="text/css" href="css/tether.min.css">
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:600" rel="stylesheet">
+    <!-- scripts for subscribe button -->
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/gsap/latest/TimelineLite.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.17.0/plugins/CSSPlugin.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.17.0/easing/EasePack.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.17.0/TweenLite.min.js"></script>
+
+    <script src="js/tether.min.js"></script>
+    <script src="js/jquery-3.2.1.min.js"></script>
+    <script src="js/bootstrap.min.js"></script>
+    <script src="js/font-awesome-4.7.0.js"></script>
+    <script src="js/scripts.js"></script>
+</head>
+
+<body>
+
+    <div class="container-fluid">
+
+        <!-- HERO SECTION -->
+        <div class="row min-full-height"
+        id="hero">
+        <div class="col-10 offset-1">
+
+            <!-- The navbar is contained within a row with id="nav-bar" -->
+            <div class="hidden-md-down row text-center text-white" id="nav-bar">
+                <div class="col-2 offset-1"><a href="https://www.hackinthenorth.com/2016/"><h4 class="nav-bar-link">HINT 2016</h4></a></div>
+                <div class="col-2"><a href="https://www.hackinthenorth.com/2017/"><h4 class="nav-bar-link">HINT 2017</h4></a></div>
+                <div class="col-2"><img src="assets/logo.png" class="nav-bar-logo"></div>
+                <div class="col-2"><a href="https://www.hackinthenorth.com/team.html"><h4 class="nav-bar-link">TEAM</h4></a></div>
+                <div class="col-2"><a href="https://www.hackinthenorth.com"><h4 class="nav-bar-link">v3.0</h4></a></div>
```

Comment:
What is the purpose of this link? Can we have something useful with this?

Reply:
HINT 2017 link can be replaced with its project showcase one.
Same can be done with 2016 one and v3.0 with the gallery.